sdk/monitor/monitor-query/src/logsQueryClient.ts

```diff
@@ -60,14 +61,16 @@ export class LogsQueryClient {
    * @param options - Options for the LogsClient.
    */
   constructor(tokenCredential: TokenCredential, options?: LogsQueryClientOptions) {
-    // This client defaults to using 'https://api.loganalytics.io/v1' as the
+    // This client defaults to using 'https://api.loganalytics.io/' as the
     // host.
-
+    const credentialOptions = {
+      credentialScopes: options?.audience
+    };
     this._logAnalytics = new AzureLogAnalytics({
       ...options,
       $host: options?.endpoint,
       endpoint: options?.endpoint,
-      credentialScopes: options?.credentialOptions?.credentialScopes ?? defaultMonitorScope,
+      credentialScopes: credentialOptions?.credentialScopes ?? defaultMonitorScope,
```

Comment:
don't you need `formatScope()` here?

Reply:
No - formatScope is used only for metricsQueryClient